solutions/dmoj-Phidias.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: dmoj-phidias
+source: DMOJ
+title: Phidas
+author: Jesse Choe
+---
+
+Let $dp[i][j]$ be the minimum residual area with a rectangular slab size of $i X j$. Initialize $dp[i][j]$ as $i * j$ and set $dp[x_i][y_i]$ as 0, where $x_i, y_i$ are the width and heights of the desired plate sizes, respectively. Check every sub-rectangle with either $i$ or $j$ fixed (but not both) and store the residual area of the optimal sub-rectangle.
```

Comment:
textttify `dp`

Reply:
thought texttt was only for the longest of variable names?

Reply:
texttt is for variable names with length greater than 1

Reply:
oh alr

Reply:
i've been super inactive lately could you clarify what u mean by "textttify"?

Reply:
he means wrap `dp` with `\texttt{}`